spec/presenters/presentation_presenter_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe PresentationPresenter do
+  subject { described_class.new(presentation, nil) }
+
+  describe '#associable_events' do
```

Comment:
Sounds good. I was thinking about testing that but my thought pattern was that the `.upcoming` method was already tested so I could skip that test. I'll make the changes.